tests/src/test/scala/com/nvidia/spark/rapids/JoinsSuite.scala

```diff
@@ -103,44 +103,43 @@ class JoinsSuite extends SparkQueryCompareTestSuite {
   }
 
   for (buildRight <- Seq(false, true)) {
-    for (buildEmpty <- Seq(false, true)) {
-      def generateLeftTable(spark: SparkSession): DataFrame = {
-        import spark.sqlContext.implicits._
-        if (buildEmpty && !buildRight) {
-          Seq[(Long, Long)]().toDF("longs", "more_longs")
-        } else {
-          longsDf(spark)
+    for (leftEmpty <- Seq(false, true)) {
+      for (rightEmpty <- Seq(false, true)) {
+        def generateLeftTable(spark: SparkSession): DataFrame = {
+          if (leftEmpty) {
+            longsDf(spark).filter("longs = 132435465768")
```

Comment:
Why is this preferable over the prior method (to get an empty dataframe)? It's a nit, the main thing you'd worry about is someone adding this specific value into `longsDf` or `biggerLongsDf` and this not being an empty df anymore.

Reply:
In theory Spark could trivially tell that the former way to build the LocalTableScan is really an empty scan, and that could trigger optimizations at query time for empty relations.  Using the filter makes it harder for the query optimizer to game the plan into not testing what we want to test.  I'll add a comment.